### ISSUE TITLE
June 2024 Meeting Topics

### ISSUE THREAD

**Issue by dragon-foobar** [MelbourneBitDevs/meetup#23]
Link anything you'd like to discuss: news, demos, ideas, whatever!

**Comment by dragon-foobar** [comment]
Victorian Bitcoin Winter Retreat Weekend
Grampians
August 23-25 
- **Camping:** 85K sats/ 100K sats (depending on Aussie dollar, approximately $90/$110)
- **Cabin Bed:** 145K sats/165K sats (approximately $130/$150)
**WIFI Available**
Workshops, Hiking, Campfire, BBQ, BitDevs(?)
**Workshops will be designed based on attendees interests**

https://vbtc.org.au/winter-retreat

**Comment by ttiiggss** [comment]
Hivetalk by bitkarrot@lnaddy.com (nostr) or [primal](primal.net/p/npub18pudjhdhhp2v8gxnkttt00um729nv93tuepjda2jrwn3eua5tf5s80a699)

[hivetalk](https://hivetalk.org)
[hivetalk on linuxunplugged podcast](
https://primal.net/e/note1wczqqz8n4ha2k54uv2he7m2dssrxl30vla0ra5swag6fd6euunvsrtlhcq)

**Comment by ttiiggss** [comment]
https://primal.net/e/note1sr6h8r2g2g730lkp64f6uev9je4l7jvlka52k6skw7m4jzkt2rxq9d94g5
by ODELL

THE NEXT WIKILEAKS WILL PUBLISH DIRECTLY TO NOSTR.

GOOD MORNING. 🫡

**Comment by CicheR** [comment]
### Inheritance scheme - Shamir Secret Sharing Scheme and Google's Inactive Account Manager based

More info in #25 

**TL/DR**: 
The basic process consists of dividing a wallet's seed (12/24 words) into 2 of 3 chunks using the [Shamir Secret Sharing Scheme](https://iancoleman.io/shamir/).	
Then use the GMail/Google account “[Inactive Account Manager](https://myaccount.google.com/inactive?pli=1)” feature to send the data to two different groups of people after N months of inactivity in the accounts.
The owner of the crypto assets will be responsible for maintaining all the Google accounts active (logging into them in less than N months regular basics), if the accounts become inactive, the inheritance process will be triggered.

**Comment by nickfarrow** [comment]
Fedimint Guardian DNS Issues
https://x.com/MutinyWallet/status/1805346636660429021

**Comment by nickfarrow** [comment]
Julian Assange is Free
https://x.com/wikileaks/status/1805388007475732646

**Comment by nickfarrow** [comment]
Ark Open Sourced
https://github.com/ark-network/

**Comment by nickfarrow** [comment]
Channel Balance Interpolation in the Lightning Network via Machine Learning
https://arxiv.org/pdf/2405.12087

**Comment by nickfarrow** [comment]
DoS: LND Onion Bomb
https://morehouse.github.io/lightning/lnd-onion-bomb/
https://x.com/roasbeef/status/1803227435233927279

**Comment by nickfarrow** [comment]
12 vs 24 words
https://x.com/robin_linus/status/1803118202764906970

**Comment by nickfarrow** [comment]
https://blog.lopp.net/fifteen-men-on-a-dead-man-s-switch/
